model/k8s/metrics.yaml

```diff
@@ -30,6 +30,28 @@ groups:
     instrument: gauge
     unit: "By"
 
+  # k8s.pod.network.* metrics
+  - id: metric.k8s.pod.network.io
+    type: metric
+    metric_name: k8s.pod.network.io
+    stability: experimental
+    brief: "Pod network IO"
+    instrument: counter
+    unit: "By"
+    attributes:
+      - ref: system.device
```

Comment:
Is this the equivalent of `interface`?

Reply:
Yes, that is designed based on the [hostmetricsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/networkscraper/documentation.md#systemnetworkio) which will also need to be updated to the latest SemConv once the System metrics are stable.

Reply:
However, I'm not really sure if that's a good and settled decision right now: https://github.com/open-telemetry/semantic-conventions/issues/308#issuecomment-2391638296

Reply:
If https://github.com/open-telemetry/semantic-conventions/pull/1492 makes it, we will use `network.interface.name` here.

Reply:
Changed to `network.interface.name`: [804d91a](https://github.com/open-telemetry/semantic-conventions/pull/1427/commits/804d91a851a0184d830b881beb47ac2b95e2d66a)

Reply:
Seems, this is covered. I'm going to resolve this conversation.